gzip/Dockerfile.bug

```diff
@@ -46,6 +53,8 @@ RUN wget  -nv http://repairbenchmarks.cs.umass.edu/ManyBugs/scenarios/${SCENARIO
 RUN cd src && \
     git checkout $(cat /experiment/manifest.txt) && \
     make distclean && \
+    wget https://raw.githubusercontent.com/gnutls/gnutls/master/bootstrap -O bootstrap && \
```

Comment:
Can we use a specific release of bootstrap? Currently, this will download the latest version of `bootstrap`, and so this image may behave differently if someone attempts to build it in a year's time.

Reply:
Maybe `https://raw.githubusercontent.com/gnutls/gnutls/gnutls_3_6_7/bootstrap`?

Reply:
hmmm, sure?